src/test/java/teammates/test/cases/pagedata/StudentHomePageDataTest.java

```diff
@@ -131,7 +131,8 @@ private void testFeedbackSession(int index, HomeFeedbackSessionRow row, Feedback
             String expectedPublishedStatus) {
         StudentHomeFeedbackSessionRow studentRow = (StudentHomeFeedbackSessionRow) row;
         assertEquals(session.getFeedbackSessionName(), studentRow.getName());
-        assertEquals(TimeHelper.formatTime12H(session.getEndTimeLocal()), studentRow.getEndTime());
+        assertEquals(TimeHelper.formatDateTimeForSessions(session.getEndTime(), session.getTimeZone()),
+                     studentRow.getEndTime());
```

Comment:
8-space indentation. You may need to change your IDE settings?

Reply:
IntelliJ indented it in 8 spaces, but then I looked at other assertEquals statements in the codebase just to be sure, it was indented like this only. 
Nevermind I'll change it.

Reply:
👍 The codebase is still in transition towards 8-space indentation; it hasn't been applied everywhere yet.